code/modules/items/gimmick/figurine.dm

```diff
@@ -1120,13 +1120,15 @@ ABSTRACT_TYPE(/datum/figure_info/patreon)
 	icon_state = "machine1"
 	icon_panel = "machine-panel"
 	var/sound_vend = 'sound/machines/capsulebuy.ogg'
+	var/base_icon_state = "machine1"
```

Comment:
why can't we use initial()

Reply:
in create_products the icon state was set with
```src.icon_state = "machine[rand(1,6)]"```, so initial would always have it revert to `machine1`